attachments/__init__.py

```diff
@@ -6,3 +6,9 @@ def session(*args, **kwargs):
     # Expose utils.session without importing utils from __init__ at module level.
     from .utils import session as attachment_session
     return attachment_session(*args, **kwargs)
+
+def get_AttachmentsMixin():
+    from attachments.mixins import AttachmentsMixin
+    return AttachmentsMixin
+
+AttachmentsMixin = get_AttachmentsMixin()
```

Comment:
do we need to have this in attachments.__init__?

Reply:
also __version_info__ need to be bumped to 3.2.5 or an equivalent increment to the latest attachment version in the 3.0 branch)